### PR TITLE
Rt1015 rt1020 sram size fix

### PR DESCRIPTION
**Pull request by jhqian**
sram size for RT1015 and RT1020 needs to be update.

### CODE REVIEW

File: dts/arm/nxp/nxp_rt1015.dtsi [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+#include <nxp/nxp_rt.dtsi>
+
+/* i.MX rt1015 default FlexRAM partition:
+ * ITCM: 32KB
+ * DTCM: 32KB
+ * OCRAM: 64KB
+ */
+&itcm0 {
+	compatible = "nxp,imx-itcm";
```
Comment (galak) [inline dts/arm/nxp/nxp_rt1015.dtsi]: You don't need the compatible here, its already in nxp_rt.dtsi.  Remove from all other nodes as well.

**Review by galak — DISMISSED** [review]
You don't need the compatible here, its already in nxp_rt.dtsi. Remove from all other nodes as well.

File: dts/arm/nxp/nxp_rt1015.dtsi (hunk 1 above)
Comment (jhqian) [reply dts/arm/nxp/nxp_rt1015.dtsi]: thank you, will do.

**Review by MaureenHelm — COMMENTED** [review]
Can you check that the ram size is also correct in the board yamls? I think all rt boards except rt1015 use external ram by default, so that may be the only one that needs updating.

**Comment by jhqian** [comment]
Yes, @MaureenHelm , only RT1015 is using SRAM, other RTs are using external SDRAM.

**Comment by jhqian** [comment]
It seems the min_ram defined for the two tests are not correct.

**Comment by MaureenHelm** [comment]
> It seems the min_ram defined for the two failed tests are not correct.

Will you please update them?

**Comment by jhqian** [comment]
> > It seems the min_ram defined for the two failed tests are not correct.
> 
> Will you please update them?

But I'm not sure the lowest memory it will need, since it depends on the other code for certain device. Or can I just take the size I have on RT1015 and set it to the nearest 4K? for example, if it is 33K-ish on RT1015, I'll set it to 36K?

**Comment by MaureenHelm** [comment]
> can I just take the size I have on RT1015 and set it to the nearest 4K? for example, if it is 33K-ish on RT1015, I'll set it to 36K?

Yes

**Review by MaureenHelm — APPROVED** [review]

**Comment by MaureenHelm** [comment]
A manual backport is needed for the rt1020 fix; rt1015 was added after 1.14 was released.

**Pull request merged**